brave-asynchttpclient/brave-asynchttpclient-1.9.x/src/main/java/smartthings/brave/asynchttpclient/ClientTracing.java

```diff
@@ -222,7 +222,7 @@
 
   private static final class HttpAdapter extends HttpClientAdapter<Request, HttpResponseStatus> {
 
-    @Override public boolean parseServerAddress(Request req, Endpoint.Builder builder) {
+    @Override public boolean parseServerIpAndPort(Request req, Endpoint.Builder builder) {
```

Comment:
you can remove this method entirely I think. then separately parse the IP/port like this https://github.com/openzipkin/brave/blob/master/instrumentation/httpasyncclient/src/main/java/brave/httpasyncclient/TracingHttpAsyncClientBuilder.java#L86